airflow/serialization/serde.py

```diff
@@ -286,7 +286,7 @@ def _convert(old: dict) -> dict:
 
 
 def _match(classname: str) -> bool:
-    return any(p.match(classname) is not None for p in _get_patterns())
+    return any(p.match(classname) for p in _get_patterns())
```

Comment:
I prefer the `is not None` variant for explicitness. There’s not perceivable performance difference also.

Reply:
Okay, will revert.